qt_ui/windows/mission/flight/waypoints/QFlightWaypointList.py

```diff
@@ -63,8 +63,10 @@ def update_list(self) -> None:
             self.model.setHorizontalHeaderLabels(HEADER_LABELS)
 
             waypoints = self.flight.flight_plan.waypoints
+            previous_waypoint = None
```

Comment:
I'd prefer getting rid of this one, and thus the added parameters in the methods as well, and instead cache it like you did with `self.last_tot`, though I would mark them as private with an underscore as prefix, i.e. `self._last_tot`

Other than that it looks pretty good 👍

Reply:
Should be cached now!